src/box/lua/config/instance_config.lua

```diff
@@ -232,6 +232,121 @@ local function advertise_peer_uri_validate(data, w)
     return uri
 end
 
+-- Accept a comma separated list of URIs and return the first one
+-- that is suitable to create a client socket (not just to listen
+-- on the server as, say, 0.0.0.0:3301 or localhost:0).
+--
+-- See the uri_is_suitable_to_connect() method in the instance
+-- schema object for details.
+local function find_suitable_uri_to_connect(self, uris)
```

Comment:
It seems, `self` is unused and can be dropped (`luacheck` likely complains here).

Reply:
Thanks, dropped.